tests/Couchbase.EntityFrameworkCore.FunctionalTests/Query/CouchbaseQueryTestBase.cs

```diff
@@ -34,13 +39,33 @@ protected virtual bool IgnoreEntryCount
         => false;
 
     protected virtual Expression RewriteServerQueryExpression(Expression serverQueryExpression)
-        => serverQueryExpression;
+    {
+
+        if (serverQueryExpression is FromSqlQueryRootExpression fromSqlQuery && fromSqlQuery.Sql.Contains("*"))
+        {
+            string propertyString = "";
+            foreach (var item in fromSqlQuery.EntityType.GetProperties())
+            {
+                if (propertyString != "")
+                {
+                    propertyString += ", ";
+                }
+                propertyString += item.Name;
+            }
+            string sql = fromSqlQuery.Sql.Replace("*", propertyString);
+            var exp = new FromSqlQueryRootExpression(
+                fromSqlQuery.QueryProvider,
+                fromSqlQuery.EntityType,
+                sql,
+                fromSqlQuery.Argument);
+            return exp;
+        }
+        return serverQueryExpression;;
+    }
 
```

Comment:
This expands "*" to the comma-separated list of all the properties, as further in the processing there is a check that property name that does not process wild-cards.